jujugui/static/gui/src/app/components/accordion-section/accordion-section.js

```diff
@@ -0,0 +1,97 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
```

Comment:
The big license header is no longer required. `/* Copyright (C) 2017 Canonical Ltd. */` is sufficient.